website/docs/api/regions.mdx

```diff
@@ -12,6 +12,15 @@ These functions convert H3 indexes to and from polygonal areas.
 
 ## polygonToCells
 
+Each binding's version of `polygonToCells` takes as input a GeoJSON-like data
+structure describing a polygon (i.e., an outer ring and optional holes) and
+a target cell resolution.
+It produces a collection of cells that are contained within the polygon.
+
+Containment is determined by centroids of the cells, so that a partitioning
+of polygons (covering an area without overlaps) will result in
+a partitioning of H3 cells.
```

Comment:
This looks good, but we usually have the explanatory text after the code example throughout the docs, so maybe this should be moved back down?

Reply:
I had considered that. But one benefit of having it this way is that it provides clear separation between the text that changes when you click a tab, and the text that doesn't.

Reply:
Well... then we probably need to expand the scope of this PR to re-organize all of the explanatory text along those lines (unchanging explanation above, per-language explanation below) :/

Reply:
Yeah, I can do that. Does everyone like the new organization I have above?